static/js/admin/src/MoveDropLocation.js

```diff
@@ -32,13 +32,25 @@ define('MoveDropLocation', [ 'jquery', 'jquery-ui', 'ConfirmationDialog'],
 				// Verify that it is the correct type of element and retrieve metadata
 				var metadata = self.options.dropTargetGetDataFunction($dropTarget);
 				if (!metadata) return false;
-				
+
+				var destTitle = self._formatTitle(metadata.title);
+				var targetAdminUnit = self._getAdminUnit(metadata.objectPath);
+				var adminUnitsSize = 1;
+
 				// Check that we are not moving an object to itself
 				try {
 					$.each(self.manager.dragTargets, function() {
 						if (this.pid == metadata.id) {
 							throw "Invalid destination.  Object " + this.pid + " cannot be move into itself.";
 						}
+
+						// Check if the object is being moved to another admin unit
+						var currentAdminUnit = self._getAdminUnit(this.metadata.objectPath);
+
+						if (currentAdminUnit !== undefined && targetAdminUnit !== undefined &&
```

Comment:
I haven't verified this, but I am guessing that if you move two items from the same admin unit to a different admin unit, this will result in an adminUnitsSize of 2? Which could be a bit confusing to users, particularly if they are moving a lot more items.

Should add a test to verify this use case.